.bazelci/postsubmit.yml

```diff
@@ -257,16 +257,52 @@ tasks:
       - test
   rbe_ubuntu1604:
     shell_commands:
-      - sed -i.bak -e 's/^# android_sdk_repository/android_sdk_repository/'
-        -e 's/^# android_ndk_repository/android_ndk_repository/'
-        -e 's/^# rbe_autoconfig/rbe_autoconfig/'
-        -e 's/^# load("@bazel_toolchains/load("@bazel_toolchains/' WORKSPACE
+      - sed -i.bak
```

Comment:
Why are there changes in the postsubmit and presubmit? We should get these reviewed by someone on who knows them better than I do.

Reply:
I included these commits to fix the failing presubmit jdeps_test as mentioned in #12685

Reply:
They should be submitted (and reviewed) separately, since it's not directly part of fixing this issue.

Reply:
They are already submitted, rebasing the pull request should fix this, right?

Reply:
Oh, I see. Yes, rebase onto master and that should be fine.